src/main/assisted-installer-controller/assisted_installer_main.go

```diff
@@ -83,7 +84,13 @@ func main() {
 
 	var kc k8s_client.K8SClient
 	if !Options.ControllerConfig.DryRunEnabled {
-		kc, err = k8s_client.NewK8SClient("", logger)
+		// in case of sno we want controller to start as fast as possible,
+		//in that case we want to use kubeconfig from filesystem and not pulling it from api
```

Comment:
Note that the bootstrap kubeconfig get deleted once the bootstrap process is complete

Reply:
Yep though object is already created and everything looks fine